ui/raidboss/data/06-ew/trial/golbez-ex.ts

```diff
@@ -1,14 +1,225 @@
+        if (data.terrastormCombatantDirs.length < wanted)
+          return;
+
+        const meteors = data.terrastormCombatantDirs;
+
+        data.terrastormCombatantDirs = [];
+        ++data.terrastormCount;
+
+        const dirs: { [dir: number]: string } = {
+          0: 'nw',
+          2: 'ne',
+          4: 'se',
+          6: 'sw',
+        };
+
+        for (const meteor of meteors) {
+          delete dirs[meteor];
+        }
+
+        const dirOutputs: string[] = [];
+
+        for (const dir of Object.values(dirs)) {
+          dirOutputs.push(output[dir]!());
+        }
+
+        return dirOutputs.join('/');
+      },
+      outputStrings: {
+        nw: Outputs.dirNW,
+        ne: Outputs.dirNE,
+        sw: Outputs.dirSW,
+        se: Outputs.dirSE,
+      },
+    },
+    {
+      id: 'GolbezEx Lingering Spark Bait',
+      type: 'StartsUsing',
+      netRegex: { id: '8468', source: 'Golbez', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Bait Circles',
+          de: 'Kreise ködern',
+          fr: 'Déposez les cercles',
+          ja: 'ゆか誘導',
+          cn: '集合放圈',
+          ko: '장판 유도',
+        },
+      },
+    },
+    {
+      id: 'GolbezEx Lingering Spark Move',
+      type: 'Ability',
+      netRegex: { id: '8468', source: 'Golbez', capture: false },
+      response: Responses.moveAway(),
+    },
+    {
+      id: 'GolbezEx Phases of the Blade',
+      type: 'StartsUsing',
+      netRegex: { id: '86DB', source: 'Golbez', capture: false },
+      response: Responses.getBackThenFront(),
+    },
+    {
+      id: 'GolbezEx Binding Cold',
+      type: 'StartsUsing',
+      netRegex: { id: '84B3', source: 'Golbez', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'GolbezEx Void Meteor',
+      type: 'StartsUsing',
+      netRegex: { id: '84AD', source: 'Golbez', capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'GolbezEx Black Fang',
+      type: 'StartsUsing',
+      netRegex: { id: '8471', source: 'Golbez', capture: false },
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'GolbezEx Abyssal Quasar',
+      type: 'StartsUsing',
+      netRegex: { id: '84AB', source: 'Golbez', capture: false },
+      alertText: (_data, _matches, output) => output.partnerStack!(),
+      outputStrings: {
+        partnerStack: {
+          en: 'Partner Stack',
+          de: 'Mit Partner sammeln',
+          fr: 'Package partenaire',
+          ja: '2人で頭割り',
+          cn: '2 人分摊',
+          ko: '2인 쉐어',
+        },
+      },
+    },
+    {
+      id: 'GolbezEx Eventide Triad',
+      type: 'StartsUsing',
+      netRegex: { id: '8480', source: 'Golbez', capture: false },
+      alertText: (_data, _matches, output) => output.rolePositions!(),
+      outputStrings: {
+        rolePositions: {
+          en: 'role positions',
```

Comment:
This set of output strings was copy/pasted from WoL ex, so should it be corrected there or should this trigger match the existing trigger?

Reply:
Uhhhhh.  maybe yeah? Mostly I was just trying to be consistent here in capitalization among outputs.  I think most triggers capitalize text.  I think an exception could be like an `infoText` aside like `(role positions)` or something.  I don't feel super strongly especially since css is going to capitalize it.  We could fix up the other one too.